Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx

```diff
@@ -53,6 +53,7 @@
 #include <vtkTextProperty.h>
 
 // VTKSYS includes
+#include <vtksys/Directory.hxx>
```

Comment:
Thinking about this is actually a nice one @lassoan, I wouldn't call it trivial :)